build-system/pr-check/build-targets.js

```diff
@@ -252,6 +218,15 @@ function determineBuildTargets(fileName = 'build-targets.js') {
         .join(', ')
     )
   );
+  // Test the runtime for babel plugin changes.
+  if (buildTargets.has('BABEL_PLUGIN')) {
+    buildTargets.add('RUNTIME');
+  }
+  // Test all targets on Travis during package upgrades.
+  if (isTravisBuild() && buildTargets.has('PACKAGE_UPGRADE')) {
```

Comment:
nit: is `isTravisBuild()` necessary?

Reply:
Yes, it is. For local `gulp pr-check`, it's neither desirable nor possible to run all tests (e.g. sauce / percy credentials, etc.)